src/App.js

```diff
@@ -113,25 +113,24 @@ const theme = createMuiTheme({
 
 const githubURL = 'https://raw.githubusercontent.com/FSHSchool/FSHOnline-Examples/main/';
 const log = console.log; //eslint-disable-line no-unused-vars
-let consoleMessages = [];
+const defaultInfoMessage = 'There are no messages to display in the console.';
+const defaultProblemMessage = 'There are no problems to display in the console.';
+let infoMessages = [defaultInfoMessage];
+let problemMessages = [defaultProblemMessage];
+let problemCount = 0;
 let exampleMetadata = {};
-let errorString = '';
-let warningString = '';
-let errorCount = 0;
-let warningCount = 0;
 console.log = function getMessages(message) {
-  consoleMessages.push(message);
   if (message && (message.startsWith('error') || message.startsWith('warn'))) {
-    if (message.startsWith('error')) errorCount++;
-    else warningCount++;
-  }
-  if (errorCount > 0) {
-    errorString = `${errorCount} Error`;
-    if (errorCount !== 1) errorString += 's';
-  }
-  if (warningCount > 0) {
-    warningString = `${warningCount} Warning`;
-    if (warningCount !== 1) warningString += 's';
+    if (problemMessages[0] === defaultProblemMessage) {
+      problemMessages = [];
+    }
+    problemMessages.push(message);
+    problemCount++;
```

Comment:
Was the intent to completely sort errors and warnings out of the normal console, or was the problems tab meant to just focus on _only_ the errors and warnings. I'm thinking the console would still include all input, and the problems tab would specifically have the errors. @jafeltra would know better than me, what do you think?

Reply:
The mockup was a bit ambiguous on this, but I do think it makes sense to include _all_ messages in the generic console and then simply include only the problems in the `Problems` console. I've added the error and warning back in commit `4388dc0`.